modules/installation-creating-azure-dns.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-creating-azure-dns_{context}"]
+= Creating networking and load balancing components in Azure
+
+You must configure networking and load balancing in Microsoft Azure for your
+{product-title} cluster to use. One way to create these components is
+to modify the provided Azure Resource Manager (ARM) template.
+
+[NOTE]
+====
+If you do not use the provided ARM template to create your Azure infrastructure,
+you must review the provided information and manually create the infrastructure.
+If your cluster does not initialize correctly, you might have to contact Red Hat
+support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure an Azure account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VNet and associated subnets in Azure.
+
+.Procedure
+
+. Copy the template from the *ARM template for the network and load balancers*
+section of this topic and save it as `03_infra.json` on your computer. This
+template describes the networking and load balancing objects that your cluster
+requires.
+
+. Create the deployment by using the `az` CLI:
++
+----
+$ az group deployment create -g ${RESOURCE_GROUP} \
+  --template-file "03_infra.json" \
+  --parameters privateDNSZoneName="${CLUSTER_NAME}.${BASE_DOMAIN}" \ <1>
+  --parameters baseName="${INFRA_ID}"<2>
+----
+<1> The name of the private DNS zone.
+<2> The base name to be used in resource names; this is usually the cluster's Infra ID.
+
+. Create an `api` DNS record in the public zone for the API public load
+balancer. The `${BASE_DOMAIN_RESOURCE_GROUP}` variable must point to the
+resource group where the public DNS zone exists.
+
+.. Export the following variable:
++
+----
+$ export PUBLIC_IP=`az network public-ip list -g ${RESOURCE_GROUP} --query "[?name=='${INFRA_ID}-master-pip'] | [0].ipAddress" -o tsv`
+----
+
+.. Create the DNS record in a new public zone:
```

Comment:
I would like to see examples here. My understanding is that the wording could be clearer. Seeing the commands with the real values helped me to figure out what command corresponded with the description.